db/migrate/20180126101234_add_unconfirmed_email.rb

```diff
@@ -0,0 +1,16 @@
+# as seen here: https://github.com/plataformatec/devise/wiki/How-To:-Add-:confirmable-to-Users
+class AddUnconfirmedEmail < ActiveRecord::Migration[5.1]
+  def up
+    add_column :users, :confirmation_token, :string
```

Comment:
Could you use a change_table here instead? That way you can specify the column type via the method name [1](http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_table).

Reply:
Yes I could. I see no reason why I should. You can do if you want to